### PR TITLE
51 like endpoint

### PR DESCRIPTION
**Pull request by therearedoors**
Implemented 'like' and 'unlike' endpoints (create and delete)

Refactored controllers - moved functions to separate Like and Comment files

### CODE REVIEW

**Comment by alsyoufomar** [comment]
u will need to resolve the conflicts

**Comment by therearedoors** [comment]
> u will need to resolve the conflicts

Had to delete main posts.js controller file as it's now redundant.

**Review by alsyoufomar — APPROVED** [review]

File: src/controllers/posts/likes.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  const { postId } = req.params
+  const { id } = req.user
+  try {
+    const likeOnPost = await dbClient.post.findUnique({
+      where: { id: parseInt(postId) }
+    })
+    if (!likeOnPost) {
+      return sendDataResponse(res, 404, { error: 'Post not found' })
+    }
+    await dbClient.postLike.create({
```
Comment (mikemherron) [inline src/controllers/posts/likes.js:14]: I can't see an update for the prisma schema file in this PR - was that part of another PR?

**Review by mikemherron — APPROVED** [review]
Looks good - just a question on where the schema update is?

File: src/controllers/posts/likes.js (hunk 1 above)
Comment (therearedoors) [reply src/controllers/posts/likes.js:14]: Yeah from this PR yesterday:
https://github.com/boolean-uk/team-dev-server/pull/58

**Pull request merged**